localization/english/event_localization/wc_scourge_invasion_l_english.yml

```diff
@@ -4,4 +4,16 @@
  WCCSC_5_OPT_A_death:0 "All are equal in death."
  WCCSC_5_OPT_A_legion:0 "This is your last chance, [nerzhul.GetFirstName]. Serve, or be destroyed."
  WCCSC_5_OPT_A_evil:0 "Death has come into play at last..."
- WCCSC_5_OPT_A_neutral:0 "[ROOT.Char.GetFaith.HighGodName|U] save us all!"
\ No newline at end of file
+ WCCSC_5_OPT_A_neutral:0 "[ROOT.Char.GetFaith.HighGodName|U] save us all!"
+ 
+ WCCSC_15_TITLE:0 "Jailors of the Damned"
+ WCCSC_15_DESC:0 ""Did you think of yourself shrewd enough to get rid of us, mortal?" [scourge_jailor.GetFirstName] asked. A hint of snark in [scourge_jailor.GetHerHis] voice as [scourge_jailor.GetHerHis] fiery eyes laid on you. Angered, powerless is how you felt.\n\nYou'd somehow manage to get rid of one of [scourge_jailor.GetHerHis] brothers, perhaps in a foolish bid to escape the gilded cage in which the Legion trapped you. Perhaps you should've stuck with doing what you were told to do, but there you were: powerless. As the demonic being tore the Frozen Throne apart, you pondered what could have been if you had succeeded."
```

Comment:
> "Did you think of yourself shrewd enough to get rid of us, mortal?"

I don't think the dread lord would address the "Lich King" as an Mortal, even as an insult, they are "dead" after all. I dunno how this would work if someone else was the lich king but I think something more like 
"Did you think of yourself shrewd enough to get rid of us, you a damned soul/ you a mere damned soul/you a mere puppet of death?"
or something like that

Reply:
> "Did you think of yourself shrewd enough to get rid of us, you a damned soul/ you a mere damned soul/you a mere puppet of death?"

Demons referring to the soul sound weird. Their alternative name is twisted souls.

I changed it to orc in a4dab95e848d1fba9d2b4f58290cdcf02f587e19.

Reply:
Changing it to orc does sound better, but why would a demon speaking to a soul sound weird? they literately devours souls and such? And soul could be changed to Spirit if that sound better.

But orc "works" for me

Reply:
> Changing it to orc does sound better, but why would a demon speaking to a soul sound weird? they literately devours souls and such? And soul could be changed to Spirit if that sound better.

First, it's too long for an insult. It sounds much more natural when it's just one word.
Second, demon souls are damned in another way. So it sounds like they arguing who is a more damned soul here.

Reply:
> Second, demon souls are damned in another way. So it sounds like they arguing who is a more damned soul here

I get that, but i would assume don't see themself as damned but as elevated? I do think the "insult" should be of what Ner'zul have "become" and not of what he was, because I don't think ner'zul view himself as an orc anymore even if he wish he was.

I get wanting just one word for an insult, i would prefer if it was in tone of him being a spirit or eccence of death, but that's hard to describe with one word.
What if he called him "Did you think of yourself shrewd enough to get rid of us, Lich?" then? why i don't think it is "very" insulting it describe what he is more then mortal or orc in my opinion?

Reply:
> I do think the "insult" should be of what Ner'zul have "become" and not of what he was, because I don't think ner'zul view himself as an orc anymore even if he wish he was.

That's a dreadlord line, no Ner'zhul's.

> What if he called him "Did you think of yourself shrewd enough to get rid of us, Lich?"

Liches are more powerful than orcs so it doesn't work as an insult.

Reply:
I do understand that but once you are just a soul your neither mortal nor an orc, and i think even the dreadlords get that (coming from the shadowlands and all) what about just calling him undead then? or failed shaman or something? 

I do think calling him a puppet in some way would be the most insulting and in line of what dreadlords think of undead

> While the nathrezim are competent necromancers, they were derided by Amal'thazad, who said they were merely making "puppets of bone" when trying to recreate the masterpiece that was the Scourge

So while it's alich saying that, it true that the dreadlords did see the scourge as puppets or pawn for them to use, so the insult should probably be about something on how Ner'zul/lich king is a puppet or a tool.

"Did you think of yourself shrewd enough to get rid of us, puppet?"
"Did you think of yourself shrewd enough to get rid of us, imprisoned pawn?"
"Did you think of yourself shrewd enough to get rid of us, No life King?" (a insulting way to call an undead, a reference from Hellsing)
"Did you think of yourself shrewd enough to get rid of us, undead?"
"Did you think of yourself shrewd enough to get rid of us, spirit?"

while no one of these are great, i do think Undead or Spirit is better then Mortal or Orc, and still sound insulting, because as you said he is a great being a lich king and calling him something else is demeaning

Reply:
I read these variants and still I consider orc or mortal the most fitting. Refering to race sounds natural. In Ner'thul's case, refering to his mortal origin.

Reply:
How about just calling him by his name then? 
"Did you think of yourself shrewd enough to get rid of us, Ner'zul?"

Reply:
Calling by name isn't insulting. The text isn't mine. I wouldn't change it that much. I'm fine with it being either mortal or orc.

Reply:
well, I think calling "mortal" is very unfitting and unintelligent insult made by a dreadlord, since no undead are mortals (you have to be alive to be mortal) and usually you don't go around calling ghosts and spirit for humans, especially those who is considered to have become "monstrous" or "divine".

Calling someone by their name can be insulting, especially if they have a rank that is "above" someone or who have a rank where you normally address them by their title, but that's more about how ones culture works.

I think "orc" works, it's definitely better then "mortal" but i still think it's a bad insult for the "Spirit" that claim rule over all undeath & death on an entire world. 

But since none of my other example's seem to be better then an insult calling a spirit "alive" I'll go ahead and approve, because i don't think i can come up with something more acceptable

Reply:
> I think calling "mortal" is very unfitting and unintelligent insult made by a dreadlord, since no undead are mortals

He is born mortal.

Reply:
And so was the majority of demons, and many of the void beings. Just because i was a child at one point of my life doesn't mean i am now. And again being born a mortal doesn't change the fact that he is dead/a spirit.

**Mortal** means susceptible to death; the opposite of immortal.
**Immortality** is eternal life, being exempt from death; unending existence.

While "true" immortality might not exist in warcraft, Ner'zul is very much as "Immortal" as any "Demons" would be be, undead are literally not mortals, mortals are always "alive".  he cannot be killed cause he is already dead, but he can be bannished away from azeroth (to the shadowlands) or is "spirit" might be drained, or destroyed, but i guess that's the cause for demons to

Reply:
> And so was the majority of demons, and many of the void beings. Just because i was a child at one point of my life doesn't mean i am now. And again being born a mortal doesn't change the fact that he is dead/a spirit.

Dreadlords born immortals. They may consider mortals lesser beings.

> Ner'zul is very much as "Immortal" as any "Demons" would be be

Nah, demons have immortal souls, that's way cooler.

Reply:
lol, they might be more "invulnerable" then Ner'zul and the common undead, but both are very much "immortal" in the case that "mortality" in unaffecting them, the main differance in demons, or dreadlord in this case, is that they are... "free"? their will are not "bound" to something, while their soul might be bound to the twisting neither, they still posses "choice" just like varimathras betrayed his brother and the legion (to my not-yet finished shadowlands lore) while most undead are bound to necromancers, some necromantic magic or ner'zul with some exceptions (like the forsaken).

Wowpedia:
> The **undead** are **former** mortals who have died and have had their **souls** trapped between life and death.

Zumbak:
> Dreadlords born immortals. They may consider mortals lesser beings.

> Nah, demons have immortal souls, that's way cooler.

So i while i don't disagree, Dreadlords would definitely consider undeads "weaker beings" or lesser souls/spirit, tools & pawn. And that Demons are "coolers" that does not mean that undeads are "mortals" at the very least.

Reply:
> both are very much "immortal"

Ner'zhul is born mortal. They're not the same. It could be weird if it's one generic undead orc calling another generic undead orc mortal. Or another generic undead calling the Lich King mortal.

Reply:
Ner'zul as the lich king is very much an "immortal" being. whatever that is the "same" immortality as a demon i can agree that it's not.
what? yes undead calling other undead "mortals" is very weird, that was my point. 
**Mortals** = can die 
**Undead** = can't die, their souls is stuck in limbo/trapped between life and death. Or Undead=Unmortal=Immoryal=opposite of being mortal. And while most "Dreadlords" might have been born in the shadowlands as death spirits, Kil'jaeden & Archimonde where once "mortals" but they are very much demons now. They "acended" for lack of a better word into demonhood. Mortals don't become "undead" when they die, they "die" and have their souls acend to their afterlife, Undead are not "shadowlands" inhabitant (unless shadowlands contradicts this) they are beings between what a Shadowland being is and what an "alive on the normal plane being" is, most venthyr's was mortal, but i doubt they consider themselves "mortals". they are another being now, just as "man'ari eredar" are an other being from when they are alive. Undead are not mortals, they are consider monstrous beings, a mockery of life or "mortality" No human would ever call an zombie or lich for "mortal" nor would a demon because that's just weird.

Reply:
> Ner'zul as the lich king is very much an "immortal" being.

Ner'zhul isn't born immortal. They are not. Thus dreadlords may call him mortal. That's my point.

When the dreadlord says Ner'zhul is mortal, he means he's born mortal, lived for about 80 years, and no match for him, a creature of the age of the universe itself.

> Just because i was a child at one point of my life doesn't mean i am now.

Using your example, 60 year old may call you kiddo while you're in your 30. Is it true and you're a kid? No. That's an insult referring to your age difference.

Let's stop this conversation already.

Reply:
Moreover, the current "orc" word refers to the same differences: age gap and immortality gap. Orcs are indeed mortals. That's why I agreed to change it. It doesn't change the meaning of the insult.

Reply:
I already agreed to orc, so I agree that we should stop to, "but" 

> Ner'zhul isn't born immortal. They are not. Thus dreadlords may call him mortal. That's my point.

I get this, but it makes zero sense as an insult, Ner'zul didn't want to become "undead" being undead is "way" worse then being mortal, being mortal or a demon is probably way better then being undead. For while i assume say dreadlords can take pleasure in their "deeds" being undead is a state of "negativity" for while Ner'zul might have taken a "liking" to be the "ruler" of undeath, it is probably not the "state of being" he sought to become as an "orc"

calling someone "alive" which what "mortal" means is a very weird insult of an Undead. at best it would make Ner'zul "sad" because he is mortal/alive no longer, but i doubt that, he probably very much have accepted his new status as undead. Undead would be below Demon, calling him "Undead" as a demeaning term as pawn/puppet or as "lower form of being"would make much more sense then calling him "alive" to show him that they are ancient and he just a young upstart whom just started to taste power.

Again, i understand what is trying to be said by "mortal" and "orc" i'm just saying that it doesn't have that affect because it seams illogical as an insult, because it doesn't have the impact for the insult's intention, in my opinion.

But as i said, _i've approved the pull_, i can accept that my "examples" was not good enough, but you can't convince me calling an undead being/monster alive or hinting to it's prior existence is a good insult that shows the age difference. 
At best it's a mockery to that he "ain't" alive or an orc any more, and is why i can accept the choice or word.

Reply:
What about "puppet".

Reply:
Scroll up, I already proposed that :P

Reply:
> you can't convince me calling an undead being/monster alive

I don't try to convince you, I try to close the topic. I won't change it from the mortal or orc because it's not my text and both its author and I approve the wording so it's 2 vs 1.

Reply:
I listened to you, changed it to the orc as a consensus but you keep talking about the mortal like it's still the case.